transactions/src/main/resources/ehcache-tx-ext.xsd

```diff
@@ -17,13 +17,15 @@
 
 <xs:schema version="1.0"
            xmlns:tx="http://www.ehcache.org/v3/tx"
+           xmlns:eh="http://www.ehcache.org/v3"
            xmlns:xs="http://www.w3.org/2001/XMLSchema"
            elementFormDefault="qualified"
            targetNamespace="http://www.ehcache.org/v3/tx">
+  <xs:import namespace="http://www.ehcache.org/v3"/>
 
-  <xs:element name="xa-store" type="tx:xa-store-config-type"/>
+  <xs:element name="xa-store" type="tx:xa-store-config-type" substitutionGroup="eh:service-configuration"/>
```

Comment:
Does these changes enable any simplification in `TxCacheManagerServiceConfigurationParser` and/or `TxCacheServiceConfigurationParser`?  (not that much is possible ...)